content/en/docs/3.features/7.loading.md

```diff
@@ -25,12 +25,14 @@ export default {
 }
 ```
 
+::alert{type="info"} If you do not see the loading bar when you move between routes, the time to load the page is short enough for the users to ignore. If you want the loading bar to appear even if the time is short, try `throttle: 0`. ::
+
```

Comment:
The content of the `alert` component must be on its own line or it won't be properly interpreted. Like this:
```
::alert{type="info"}
If you do not see the loading bar when you move between routes, the time to load the page is short enough for the users to ignore. If you want the loading bar to appear even if the time is short, try `throttle: 0`.
::
```
See the deploy preview: https://deploy-preview-2059--nuxt.netlify.app/docs/features/loading

Reply:
I fixed the issue. 

Thanks :)

I was not aware of it when I looked at the examples for alert in other pages.

Reply:
Oh right, if you find other places where MarkDown Components are broken, feel free to tell us even a quick issue 🙂